dprint.json

```diff
@@ -21,7 +21,9 @@
     "**/pnpm-lock.yaml",
     "external-crates/move/crates/move-stdlib/**/docs/**/*.md",
     "nre/helm/graphql/templates/**/*.yaml",
-    "nre/helm/indexer/templates/**/*.yaml"
+    "nre/helm/indexer/templates/**/*.yaml",
+    "crates/iota/src/unit_tests/fixtures/upgrade_errors/malformed_move_toml/starts_second_line/Move.toml",
+    "crates/iota/src/unit_tests/fixtures/upgrade_errors/missing_module_toml/emoji/Move.toml"
```

Comment:
Sure about this?

Reply:
Yes, for the first file it was removing the first empty line and for the second file it was printing an error

Reply:
Maybe we should exclude all fixtures?

Reply:
Wouldn't do that, it's an exception that they are invalid

Reply:
Yeah but presumably we don't really want to touch any of these with a formatter ever

Reply:
I guess you're right, is also more future proof, changed in https://github.com/iotaledger/iota/pull/5825/commits/44b933a2667924f2d04a1461785af8e3a94eb26b